scripts/process_new_format.py

```
@@ -171,65 +180,223 @@ def process_markdown_faq_data(markdown_faq_data: dict[str, str]) -> None:
+                    )
+                    current_question_html_str = None
+                else:
+                    rulings_for_item.append(
+                        Ruling(
+                            source_card_code=source_card_code,
+                            ruling_type=ruling_type_enum,
+                            text=full_content_plain_text,
+                            provenance=current_provenance,
+                            original_html_snippet=original_snippet_html,
+                            related_card_codes=related_codes
+                        )
```

Ruff (0.11.9)

Indentation is not a multiple of 4

(E111)